frontend/src/metabase/admin/performance/utils.unit.spec.ts

```diff
@@ -85,15 +85,15 @@ describe("scheduleSettingsToCron", () => {
       schedule_hour: 23,
     };
     const cron = scheduleSettingsToCron(settings);
-    expect(cron).toEqual("0 5 23 15 * ?");
+    expect(cron).toEqual("0 5 23 15 * ? *");
   });
 
   it("missing minute and hour should default to wildcard", () => {
     const settings: ScheduleSettings = {
       schedule_type: "daily",
     };
     const cron = scheduleSettingsToCron(settings);
-    expect(cron).toEqual("0 * * * * ?");
+    expect(cron).toEqual("0 * * * * ? *");
```

Comment:
Is this syntax change compatible with the schedules already recorded in users' app dbs?

Reply:
Yes, BE uses Quartz which understands both 6-part and 7-part cron expression. Most places on UI used 7-part one (well, 2 out of 3). So I've decided to make it uniform.

If you look at `handleScheduleChange` method of `ModelCachingScheduleWidget`, conversion to 7-part is quite brutal, but I didn't want to risk refactoring it. 

As for `scheduleSettingsToCron` function, it is used to build a request for server, not for parsing existing strings. So everything should be good here.